src/TimeseriesComponent/TimeseriesComponent.js

```diff
@@ -562,21 +588,25 @@ class TimeseriesComponent {
         const clip = g.append('g').attr('class', 'timeseries-clip');
         chartRoot = clip.append('g').attr('class', 'timeseries-chart');
       } else {
-        g.selectAll('.timeseries-data,.x-axis,.timeseries-line').remove();
+        g.selectAll('.timeseries-data,.x-axis,.x-grid-axis,.timeseries-line').remove();
```

Comment:
Where do the `x-...` classes come from?

Reply:
That's the classes we set when rendering the axes/components.